src/routes/routes.tsx

```diff
@@ -43,6 +44,10 @@ export const Routes = () => {
     (selectedMyUser && selectedMyUser.role !== 0) || hasAdminView
       ? AdminPage
       : DefaultPage;
+  const SneakyAdminPage1 =
```

Comment:
I confused myself with this naming,
We allow super users to see more pages.

I'll probably end up rewriting some of these things as I add tokens authentication. I think we will want to find a way to merge intermediate work